### DISCUSSION TITLE
help changing default image editing thing

### DISCUSSION THREAD

**Discussion by patritha** [FossifyOrg/General-Discussion#402] [Q&A] [answered]
i fat fingered the always button after pressing the edit button on an image. i dont want to use the app i pressed always on and nothing i do lets me change it. HELP!

i am on android 14

**Comment by Aga-C** [comment, marked as answer]
Go to the app info in system settings (e.g. through long-pressing the app icon). There's `Open by default` - go there and tap `Clear default preferences`.

**Reply by patritha** [reply]
you are a real one, thanks!
